main/windows/main.js

```diff
@@ -80,6 +92,9 @@ function init () {
 }
 
 function dispatch (...args) {
+  if (args[0] === 'saveState') {
```

Comment:
If we choose to go with dispatching the bounds (otherwise see my other comment), I think we should pass the bounds directly in `main/index.js` like this: ` windows.main.dispatch('saveState', windows.main.win.getBounds())`, or create a separate method to avoid special cases in the `dispatch` method.